src/player.rs

```diff
@@ -1,5 +1,5 @@
 use bevy::{prelude::*};
-use crate::backgrounds::{Background, TILE_SIZE, LEVEL_LENGTH, LEVEL_HEIGHT};
```

Comment:
Wait wait wait, why were these removed?

The PR to change player movement was not correctly reviewed, with this code the player can move well past world boundaries without being stopped.

Shouldn't we put these back and put a check in `move_player`?